packages/core/src/connectivity/scp-cf/destination/destination-from-service.ts

```diff
@@ -80,13 +84,16 @@ class DestinationFromServiceRetriever {
   ): Promise<Destination | null> {
     const subscriberToken =
       await DestinationFromServiceRetriever.getSubscriberToken(options);
+
+    const xsuaaCredentials = getXsuaaServiceCredentials(options.userJwt);
```

Comment:
Not related to this PR, but seems important: When I added support for x509, I updated the `ServiceCredentials` but not the `XsuaaServiceCredentials`... Shouldn't this support x509 as well?!